Update autoinstallation copy for containerized server (#3047)
uyuni-project/uyuni-docs: modules/client-configuration/pages/autoinst-distributions.adoc

The autoinstallation process relies on an several files to initiate the installation.
These files include the Linux kernel, an initial RAM disk, and other files required to boot the operating system in installation mode.

> {productname} use [systemitem]``mgradm`` tool to copy installation file from the source to the server container.

Reviewer: ```suggestion
{productname} uses the [systemitem]``mgradm`` tool to copy the installation file from the source to the server container.
```

.Procedure: Importing Files from Installation Media

> . Use [systemitem]``mgradm`` tool to import installation data from the ISO

Reviewer: ```suggestion
. Use [command]``mgradm`` to import installation data from the ISO:
```

+
----
# mgradm distribution copy <image_name>.iso <image_name>
----
+

> . Take a note of the distribution path reported by [systemitem]``mgradm`` tool
>   You will need it when you declare the distribution to {productname}.

Reviewer: ```suggestion
. Take a note of the distribution path reported by [command]``mgradm``.
  You will need it when you declare the distribution to {productname}.
  
  
```

=== Distribution autodetection and registration

> [systemitem]``mgradm`` is able to automatically detect distribution name and register it to the server. Provided ISO image needs to contain [path]``.treeinfo`` file.

Reviewer: ```suggestion
[systemitem]``mgradm`` is able to automatically detect distribution name and register it to the server.
Provided ISO image needs to contain a [path]``.treeinfo`` file.
```

.Procedure: Import distribution file with autodetection and registration

> . Use [systemitem]``mgradm`` tool

Reviewer: ```suggestion
. Use [command]``mgradm``:
```

.Procedure: Importing Files from Installation Media

> . Use [command]``mgradm`` to import installation data from the ISO:

Reviewer: ```suggestion
. Use [command]``mgradm`` to import installation data from the ISO image:
```

.Procedure: Extracting Files from an Installation Package
. On the {productname} Server, install the package whose name starts with [package]``tftpboot-installation``.
  You can determine its exact name with the command [command]``zypper se tftpboot-installation``
. You can install the package to different root to avoid need for restart using following command:
+
----
# mkdir /opt/tftpinstall
# zypper --installroot /opt/tftpinstall install tftpboot-installation-SLE-Micro-5.5-x86_64
----

> . Determine where the installation files went with the command [command]``ls -d /opt/tftpinstall/usr/share/tftpboot-installation/*``.
> . Copy installation files using [systemitem]``mgradm`` tool

Reviewer: ```suggestion
. Find the installation files with the command [command]``ls -d /opt/tftpinstall/usr/share/tftpboot-installation/*``.
. Copy installation files using [command]``mgradm``:
```

+
----
# mgradm distribution copy /opt/tftpinstall/usr/share/tftpboot-installation/SLE-Micro-5.5-x86_64 SLE-Micro-5.5-x86_64
----

> . Take a note of the distribution path reported by [systemitem]``mgradm`` tool

Reviewer: ```suggestion
. Take a note of the distribution path reported by [systemitem]``mgradm`` tool.
```
